src/vue/pages/pay/AssetInformation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="asset-information">
+    <div class="asset-information__header">
+      <asset-logo-dark
+        class="asset-information__logo"
+        :asset-code="asset.code"
+        :logo-url="assetLogoUrl"
+      />
+      <div class="asset-information__info">
```

Comment:
Do not use 'info' or 'data' or something like this in class naming anywhere.

Reply:
asset-viewer__name

Reply:
renamed

Reply:
renamed